test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs

```diff
@@ -571,6 +575,93 @@ public void AnomalyDetectionWithSrCnn(bool loadDataFromFile)
             }
         }
 
+        [Theory, CombinatorialData]
+        public void TestSrCnnBatchAnomalyDetector(
+            [CombinatorialValues(SrCnnDetectMode.AnomalyOnly, SrCnnDetectMode.AnomalyAndExpectedValue, SrCnnDetectMode.AnomalyAndMargin)]SrCnnDetectMode mode,
+            [CombinatorialValues(true, false)]bool loadDataFromFile)
+        {
+            var ml = new MLContext(1);
+            IDataView dataView;
+            if (loadDataFromFile)
+            {
+                var dataPath = GetDataPath(Path.Combine("Timeseries", "anomaly_detection.csv"));
```

Comment:
>Path.Combine( [](start = 43, length = 13)

I think there is an overload of `GetDataPath` which does `Path.Combine` for you.

Reply:
done.

---
In reply to: [426440599](https://github.com/dotnet/machinelearning/pull/5135#discussion_r426440599) [](ancestors = 426440599)